GeneralsMD/CMakeLists.txt

```diff
@@ -12,5 +12,32 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 # Add main build targets
 add_subdirectory(Code)
 
+# If we are building on windows for windows, try and get the game install path from the registry.
+if("${CMAKE_HOST_SYSTEM}" MATCHES "Windows" AND "${CMAKE_SYSTEM}" MATCHES "Windows")
+    if(NOT ZEROHOUR_INSTALL_PREFIX)
```

Comment:
Should we perhaps prefix these with GENZH_ so they group better with our other options in cmake (assuming they are visible options in cmake)?

Reply:
Done